internal/service/handlers/get_airdrop.go

```diff
@@ -13,42 +13,46 @@ import (
 
 func GetAirdrop(w http.ResponseWriter, r *http.Request) {
 	var (
-		id  = chi.URLParam(r, "id")
-		err = validation.Errors{"{id}": validation.Validate(id, validation.Required)}.Filter()
+		nullifier = chi.URLParam(r, "nullifier")
+		err       = validation.Errors{"{nullifier}": validation.Validate(nullifier, validation.Required)}.Filter()
 	)
 	if err != nil {
 		ape.RenderErr(w, problems.BadRequest(err)...)
 		return
 	}
 
-	participant, err := ParticipantsQ(r).Get(id)
+	airdrop, err := AirdropsQ(r).
+		FilterByNullifier(nullifier).
+		FilterByStatus(data.TxStatusCompleted).
```

Comment:
Why do we show only completed ones? I suppose that this endpoint may be used for polling airdrop status, isn't it?

Reply:
agreed, I will try to select all, if there is completed one -- return it